<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/resource/mainicon.ico</normaloff>:/resource/mainicon.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="mainMenuStandartButton">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>110</y>
      <width>120</width>
      <height>45</height>
     </rect>
    </property>
    <property name="baseSize">
     <size>
      <width>120</width>
      <height>45</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Standart</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="mainMenuTimeButton">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>185</y>
      <width>120</width>
      <height>45</height>
     </rect>
    </property>
    <property name="baseSize">
     <size>
      <width>120</width>
      <height>45</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Time</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="mainMenuMoodButton">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>260</y>
      <width>120</width>
      <height>45</height>
     </rect>
    </property>
    <property name="baseSize">
     <size>
      <width>120</width>
      <height>45</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Mood</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="mainMenuFutureButton">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>335</y>
      <width>120</width>
      <height>45</height>
     </rect>
    </property>
    <property name="baseSize">
     <size>
      <width>120</width>
      <height>45</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Future</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="mainMenuSettingsButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>435</y>
      <width>35</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
